contracts/external/balancer/IBalancerMinter.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// See https://etherscan.io/address/0x239e55F427D44C3cc793f49bFB507ebe76638a2b
+
+pragma solidity ^0.8.4;
+
+interface IBalancerMinter {
+    event Minted(address indexed recipient, address gauge, uint256 minted);
```

Comment:
Could the `gauge` be indexed too? Useful to be able to filter both sides of the relation

Reply:
this is Balancer's external contract 😅